packages/sdk-js/src/auto-wrapper.ts

```diff
@@ -198,8 +240,20 @@ gb.setRenderer(() => {
   document.dispatchEvent(new CustomEvent("growthbookdata"));
 });
 
-// Load features/experiments
-gb.loadFeatures();
+(async () => {
+  // Hydrate sticky bucket service
+  if (stickyBucketService && windowContext.stickyBucketAssignmentDocs) {
+    for (const key in windowContext.stickyBucketAssignmentDocs) {
+      const doc = windowContext.stickyBucketAssignmentDocs?.[key];
+      if (doc) {
+        await stickyBucketService.saveAssignments(doc);
+      }
+    }
+  }
+
+  // Load features/experiments
+  gb.init({ payload: windowContext.payload });
```

Comment:
To match existing behavior, we should default to `streaming: true` here, but give people a way to opt-out via window or data context (can re-use existing `backgroundSync` setting if we want for this)